pkgs/top-level/all-packages.nix

```
@@ -21717,6 +21717,8 @@ in
 
   i3-layout-manager = callPackage ../applications/window-managers/i3/layout-manager.nix { };
 
+  i3-resurrect = python3Packages.callPackage ../applications/window-managers/i3/i3-resurrect.nix { }; 
```

```suggestion
  i3-resurrect = python3Packages.callPackage ../applications/window-managers/i3/i3-resurrect.nix { };
```

to make the editor check happy.

---

Done.